zeebe-cluster/values.yaml

```diff
@@ -13,6 +13,7 @@ partitionCount: "3"
 replicationFactor: "3"
 cpuThreadCount: "2"
 ioThreadCount: "2"
+gatewayMetrics: true
```

Comment:
@Zelldon this is a little bit contradictory.. we should have a default to false, but then we are defaulting to true. I would recommend choosing which one is the default and then not add this to the `values.yaml` file. Does this make sense?

Reply:
Yes kind of agree. I added this to the values file to have an example setting; so users can also see that they can set this property, which is per default true if you use the helm charts. But yes might be better to have this somewhere as a documentation. I set the default in the configMap to false because I expected to disable it when no value was given/set.

Reply:
@Zelldon that is alright. but let's remove the "example" from the `values.yaml` file and also make sure that you add the new property in the README.md of the chart -> https://github.com/zeebe-io/zeebe-cluster-helm/blob/master/README.md where all the available parameters specifically for this chart are documented.